collection-scripts/gather_services_status

```diff
@@ -87,6 +92,29 @@ get_heat_status() {
     ${BASH_ALIASES[os]} orchestration service list > "$HEAT_PATH"/service_list
 }
 
+# Nova service gathering - sevices, hypervisors, cells, host mappings, allocation audit
+get_nova_status() {
+    local NOVA_PATH="$BASE_COLLECTION_PATH/ctlplane/nova"
+    mkdir -p "$NOVA_PATH"
+    run_bg ${BASH_ALIASES[os]} compute service list '>' "$NOVA_PATH"/service_list
```

Comment:
nit: i know your following the exisitn pattern but ideally

"$NOVA_PATH"/service_list would be "${NOVA_PATH}/service_list"